blog/2024-07-3-uppy-4.0.md

```diff
@@ -0,0 +1,408 @@
+---
+title:
+  'New Uppy 4.0 major: TypeScript rewrite, Google Photos, React hooks, and much
+  more.'
+date: 2024-07-03
```

Comment:
The date should be 2024-07-10, when the release was pushed to npm
```suggestion
date: 2024-07-10
```
Also the name of the file needs to be amended